ui/src/dashboards/actions/index.ts

```diff
@@ -628,25 +628,21 @@ export const hydrateTempVarValuesAsync = (
     const dashboard = getState().dashboardUI.dashboards.find(
       d => d.id === dashboardID
     )
-
-    const tempsWithQueries = dashboard.templates.filter(
-      ({query}) => !!query.influxql
-    )
-
-    const asyncQueries = tempsWithQueries.map(({query}) =>
-      getTempVarValuesBySourceQuery(source, {
-        query: makeQueryForTemplate(query),
+    const templates: Template[] = dashboard.templates
+    const queries = templates
+      .filter(template => !!template.query.influxql)
+      .map(async template => {
+        const query = makeQueryForTemplate(template.query)
+        const response = await proxy({source: source.links.proxy, query})
```

Comment:
how do you feel about making a function in `ui/src/dashboards/apis/index.js` that performs this proxy call so that the function name can give more insight as to what this request is doing?

Reply:
how about renaming `proxy`? since it's only function AFAIK is to execute a InfluxQL query